extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationServiceSpec.js

```diff
@@ -165,4 +176,22 @@ describe('ItemSelectionInputValidationService', function() {
       message: 'Please ensure the choices are unique.'
     }]);
   });
+
+  it(
+    'should give a warning if number of rule inputs are lesser than 2 and ' +
```

Comment:
if the number of elements in the rule input is less than 2, and the "proper subset" rule is used.

Reply:
Done